shell/server/pre-meteor.js

```diff
@@ -210,6 +210,25 @@ function checkMagic(buf, magic) {
   return true;
 }
 
+function serveSelfTest(req, res) {
+  inMeteor(() => {
```

Comment:
But it does mean that exceptions won't bubble-up to the top and destroy the entire process, which otherwise they would do, I believe.

If I'm mistaken, let me know, but I'm gently in favor of keeping it given the above.